static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-us-pre-end-of-year.js

```diff
@@ -0,0 +1,207 @@
+                p2: 'If you read and like our reporting, help us make our future more secure.'
+            },
+            endOfYear: {
+                title: 'As 2016 comes to a close…',
+                p1: '…we would like to ask for your support. More people are reading the Guardian than ever but far fewer are paying for it. And advertising revenues across the media are falling fast. So you can see why now is the right time to ask. The Guardian\'s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.',
+                p2: 'If everyone who reads our reporting – who believes in it – helps to support it, our future would be more secure.'
+            }
+        };
+
+        var cta = {
+            equal: {
+                p3: '',
+                cta1: 'Become a Supporter',
+                cta2: 'Make a contribution',
+                url1: makeUrl(membershipUrl, membershipUrlPrefix),
+                url2:  makeUrl(contributeUrl, contributeUrlPrefix),
+                hidden: ''
+            }
+
+        };
+
+        var componentWriter = function (component) {
+            ajax({
+                url: 'https://api.nextgen.guardianapps.co.uk/geolocation',
+                method: 'GET',
+                contentType: 'application/json',
+                crossOrigin: true
+            }).then(function (resp) {
+                if ('country' in resp && resp.country === 'US'){
+                    fastdom.write(function () {
+                        var submetaElement = $('.submeta');
+                        if (submetaElement.length > 0) {
+                            component.insertBefore(submetaElement);
+                            mediator.emit(epicInsertedEvent, component);
+                            $('.contributions__epic').each(function (element) {
+                                // top offset of 18 ensures view only counts when half of element is on screen
+                                var elementInView = ElementInview(element, window, {top: 18});
+                                elementInView.on('firstview', function () {
+                                    mediator.emit(epicViewedEvent);
+                                });
+
+                            });
+                        }
+                    });
+                }
+            });
+        };
+
+        var initializer = function(track) {
+            mediator.on(epicInsertedEvent, track);
+        };
+
+        var completer = function (complete) {
```

Comment:
In the same vein as above, maybe we should call this view to be explicit and make it easy to see from the variant definition exactly what is happening on impression and success